pacai/util/probability.py

```diff
@@ -68,8 +64,8 @@ def sample(distribution, values = None):
 
     return values[i]
 
-def sampleFromCounter(ctr):
-    items = sorted(list(ctr.items()))
+def sampleFromDict(dictDistribution):
+    items = sorted(dictDistribution.items())
     return sample([v for k, v in items], [k for k, v in items])
 
 def getProbability(value, distribution, values):
```

Comment:
Do we need this method?
Can we git rid of this?
(Otherwise, this method will need a good amount of work.)

Reply:
I don't believe we need this method. The client should just call sample() directly since the method handles a dictionary instance already.

Reply:
You can remove it if we don't need it.